docs/user-guide/standard_transformer_upi.md

```diff
@@ -0,0 +1,99 @@
+# Configuring Standard Transformer for UPI Model
+
+> This guide assumes you have experience using standard transformer and  familiar with UPI contract. You can read refer to https://github.com/caraml-dev/universal-prediction-interface to get details on the contract. 
+
+There are 2 key differences in Standard Transformer when it’s deployed using UPI protocol:
+
+1. Autoload Feature
+2. Separate Output Operation for Pre-process and Post-process
+
+## Autoload Feature
+
+Autoload feature is the primary mechanism for importing values in the request payload as variables or tables into standard transformer. Previously, it is done by using JSONPath query in the HTTP mode.
+
+For example in HTTP model, if you want to declare a rating variable that should use value from user_rating field of the incoming request, then you have to declare following configuration in the input.
+
```

Comment:
Would be great to also include the request example in json format.

Request payload:
```json
{
    "user_rating": 5
}
```

Standard transformer configuration:
```yaml
- variables:
...
```

Reply:
Updated